### ISSUE TITLE
Please stop

### ISSUE THREAD

**Issue by cribbage** [alekxeyuk/Skribbl.io-Bot#31]
Nobody wants to play pictionary with a bot that just displays images from google. You're ruining the game with this shit. Stop.

**Comment by Fr33styler** [comment]
i agree, its not fun to see a program playing the game for us, we can just write the word in chat and make it a who writes faster then :/
